scalding-commons/src/test/scala/com/twitter/scalding/commons/VersionedKeyValSourceTest.scala

```diff
@@ -89,7 +96,7 @@ class VersionedKeyValSourceTest extends WordSpec with Matchers {
       .sink[(Int, Int)](VersionedKeyValSource[Array[Byte], Array[Byte]]("output")) { outputBuffer: Buffer[(Int, Int)] =>
         "Outputs must be as expected" in {
           assert(outputBuffer.size === input.size)
-          val singleInj = implicitly[Injection[Int, Array[Byte]]]
+          implicitly[Injection[Int, Array[Byte]]]
```

Comment:
yeah that's true. Got rid of this one and the other one as the test seems to pass without them.